{{cookiecutter.project_shortname}}/README.md

```diff
@@ -0,0 +1,50 @@
+# {{ cookiecutter.project_name }}
+
+Welcome to your InvenioRDM instance.
+
+## Getting started
+
+Run the following commands in order to start your new InvenioRDM instance:
+
+```console
+invenio-cli containerize
+invenio-cli demo --containers
+```
+
+The above commands first builds the application docker image and afterwards
+starts the application and related services (database, Elasticsearch, Redis
+and RabbitMQ). The build and boot process will take some time to complete,
+especially the first time as docker images have to be downloaded during the
+process.
+
+Once running, visit https://127.0.0.1 in your browser.
+
+**Note**: The server is using a self-signed SSL certificate, so your browser
+will issue a warning that you will have to by-pass.
+
+## Overview
+
+Following is an overview of the generated files and folders:
+
+| Name | Description |
+|---|---|
+| ``Dockerfile`` | Dockerfile used to build your application image. |
+| ``Pipfile`` | Python requirements installed via [pipenv](https://pipenv.pypa.io) |
+| ``Pipfile.lock`` | Locked requirements (generated on first install). |
+| ``app_data`` | Application data such as vocabularies. |
+| ``assets`` | Web assets (CSS, JavaScript, LESS, JSX templates) used in the Webpack build. |
+| ``docker`` | Example configuration for NGINX and uWSGI. |
+| ``docker-compose.full.yml`` | Example of a full infrastructure stack. |
+| ``docker-compose.yml`` | Backend services needed for local development. |
+| ``docker-services.yml`` | Common services for the Docker Compose files. |
+| ``invenio.cfg`` | The Invenio application configuratio. |
+| ``logs`` | Log files. |
+| ``static`` | Static files that need to be served as-is (e.g. images). |
+| ``templates`` | Folder for your Jinja templates. |
+| ``.invenio`` | File used by Invenio-CLI. |
+| ``.invenio.private`` | File used by Invenio-CLI. |
+
+## Documentation
+
+Do you want to learn how to configure, customize, deploy and much more. Visit
```

Comment:
```suggestion
To learn how to configure, customize, deploy and much more, visit
```

Reply:
If it's a question shouldn't it be:

`Do you want to learn how to configure, customize, deploy and much more?`

Anyway, just a super-ultra-minor :)